ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml

```diff
@@ -732,6 +732,14 @@ tests:
       INSTALL_TYPE: sno
       OCP_VERSION: "4.15"
     workflow: upi-powervs-sno
+- as: ocp-e2e-agent-sno-power
+  cron: 0 1 * * 1
```

Comment:
Can we confirm the cron timings so we don't run into lease starvation?  PowerVS leases are limited, from what I understand, so you can potentially talk with Shilpa/MikeT to get the right timing here.

Reply:
OK. let me confirm it with them.

Reply:
After talked with Shilpa/MikeT, there is no problem with the timing for ABI SNO.